util/site/build-docs.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+set -e
+
+command="build"
+
+case "$1" in
+  "build"|"build-local"|"serve"|"serve-proxy")
+    command="$1"
+    public_url="${2}"
+    public_port="${3}"
```

Comment:
These args don't have documentation or validation

Reply:
They are now documented but not validated